linux/po/appstream.pot

```diff
@@ -10,402 +10,57 @@ msgstr ""
 "Content-Transfer-Encoding: 8bit\n"
 
 #. (itstool) path: component/name
-#: org.hydrogenmusic.hydrogen.metainfo.xml.in:6
+#: org.hydrogenmusic.Hydrogen.metainfo.xml.in:6
 msgid "Hydrogen"
 msgstr ""
 
 #. (itstool) path: component/summary
-#: org.hydrogenmusic.hydrogen.metainfo.xml.in:7
+#: org.hydrogenmusic.Hydrogen.metainfo.xml.in:7
 msgid "The advanced drum machine"
 msgstr ""
 
 #. (itstool) path: description/p
-#: org.hydrogenmusic.hydrogen.metainfo.xml.in:17
+#: org.hydrogenmusic.Hydrogen.metainfo.xml.in:17
 msgid "Hydrogen is both a drum sequencer for pattern-based programming and a drum synth. Its simple yet feature-rich design makes it a versatile companion, be it for quick practice at home, professional usage as fully-fledged drum machine in your studio, or drum looper on steriods for your live performance."
 msgstr ""
 
 #. (itstool) path: description/p
-#: org.hydrogenmusic.hydrogen.metainfo.xml.in:24
+#: org.hydrogenmusic.Hydrogen.metainfo.xml.in:24
 msgid "Features:"
 msgstr ""
 
 #. (itstool) path: ul/li
-#: org.hydrogenmusic.hydrogen.metainfo.xml.in:28
+#: org.hydrogenmusic.Hydrogen.metainfo.xml.in:28
 msgid "First-class MIDI support"
 msgstr ""
 
 #. (itstool) path: ul/li
-#: org.hydrogenmusic.hydrogen.metainfo.xml.in:29
+#: org.hydrogenmusic.Hydrogen.metainfo.xml.in:29
 msgid "Highly customizable"
 msgstr ""
 
 #. (itstool) path: ul/li
-#: org.hydrogenmusic.hydrogen.metainfo.xml.in:30
+#: org.hydrogenmusic.Hydrogen.metainfo.xml.in:30
 msgid "Stable and performant"
 msgstr ""
 
 #. (itstool) path: ul/li
-#: org.hydrogenmusic.hydrogen.metainfo.xml.in:31
+#: org.hydrogenmusic.Hydrogen.metainfo.xml.in:31
 msgid "Loads of drumkits to choose from"
 msgstr ""
 
 #. (itstool) path: ul/li
-#: org.hydrogenmusic.hydrogen.metainfo.xml.in:32
+#: org.hydrogenmusic.Hydrogen.metainfo.xml.in:32
 msgid "In-depth control of instrument, sample, and note properties"
 msgstr ""
 
 #. (itstool) path: ul/li
-#: org.hydrogenmusic.hydrogen.metainfo.xml.in:33
+#: org.hydrogenmusic.Hydrogen.metainfo.xml.in:33
 msgid "High quality time-stretching and pitch shifting"
 msgstr ""
 
 #. (itstool) path: ul/li
-#: org.hydrogenmusic.hydrogen.metainfo.xml.in:34
+#: org.hydrogenmusic.Hydrogen.metainfo.xml.in:34
 msgid "Automate it using MIDI or OSC API"
 msgstr ""
 
-#. (itstool) path: description/p
```

Comment:
The merging of the metainfo and releases file broken some parts of the internationalization toolchain. The `linux/po/appstream-metainfo.its` file has to be adopt as well. Otherwise large parts of the existing translations will be lost.

Reply:
I don't see where this need changing....

Reply:
nevermind. I also had to fix the makefile to have a proper dependency.